treadmill/infra/setup/scripts/configure-master.sh

```diff
@@ -0,0 +1,23 @@
+LDAP_DC=ou=treadmill,$(echo "{{ DOMAIN }}" | sed -E 's/([a-z]*)\.([a-z]*)/dc=\1,dc=\2/g')
+LDAP_URL=ldap://{{ FREEIPA_HOSTNAME }}.{{ DOMAIN }}:1389
+ZK_URL=zookeeper://foo@TreadmillZookeeper1.{{ DOMAIN }}:2181,TreadmillZookeeper2.{{ DOMAIN }}:2181,TreadmillZookeeper3.{{ DOMAIN }}:2181
```

Comment:
it is ok for now, but we need to switch running zookeeper under s6 using admin install --run.